docs/source/demos/lead_scoring.ipynb

```diff
@@ -41,7 +41,7 @@
    "outputs": [],
    "source": [
     "lead_scoring_objective = LeadScoring(\n",
-    "    true_positives=1000,\n",
+    "    true_positives=100,\n",
```

Comment:
@bchen1116 The lead scoring docs show a lead scoring score of 0 for both searches:

![Uploading image.png…]()

Reply:
This comment is out of date, right? Resolving